package.json

```diff
@@ -18,10 +18,11 @@
     "start:automated": "webpack-dev-server --color --progress --config webpack.test.js",
     "start": "webpack-dev-server --color --progress --config webpack.dev.js",
     "example": "ws --rewrite '/* -> /dist/$1' --https",
-    "prod": "npm run build",
+    "prod": "npm run build && npm run copy",
```

Comment:
@james-simpson-ST I think if we wanna have stcard.js and stcardstyles,css in main folder we should do sth like this.